tests/integration/pilot/validation_test.go

```diff
@@ -173,6 +173,8 @@ func TestEnsureNoMissingCRDs(t *testing.T) {
 				"networking.istio.io/v1beta1/VirtualService",
 				"networking.istio.io/v1beta1/WorkloadEntry",
 				"networking.istio.io/v1beta1/Sidecar",
+				"security.istio.io/v1/AuthorizationPolicy",
```

Comment:
nit: use the `aliases` part of `r` to do this automatically; the networking stuff can be cleaned up as well (we didn't have `aliases` at the time